clogMonitor/src/components/featuredInfo/FeaturedInfo.js

```diff
@@ -5,29 +5,176 @@ import WarningIcon from '@mui/icons-material/Warning';
 import InfoIcon from '@mui/icons-material/Info';
 import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward'
 import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward'
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import HorizontalRuleIcon from '@mui/icons-material/HorizontalRule';
 
-export default function FeaturedInfo(){
+export default function FeaturedInfo({ logEvents }){
+    
+    let dateToLogsCount = new Map();
+    let Severityset = new Set();
+    logEvents.forEach(e => {
```

Comment:
I strongly recommend that you do not load or pass the entire log details table. Instead, you can make an API call for just todays and yesterdays log details